app/views/divisions/division_not_found.html.haml

```diff
@@ -0,0 +1,6 @@
+- content_for :title, 'Division not found'
+%h1= yield :title
+
+No Division found. If you entered a date, please make
```

Comment:
That's a good point. I used Division because that's what is displayed in the top menu, so I assumed people knew somehow. But it seems I am wrong 🙂. 

So far, we are using this page only for the invalid date scenario and I put a message in there to explain why. Anyway, I agree the wording is not very good. If you have any suggestions in how to improve it, that would be great.